pkg/hub/deployer/deployer.go

```diff
@@ -359,7 +359,7 @@ func (deployer *Deployer) populateBasesAndLocalizations(sub *appsapi.Subscriptio
 
 		baseTemplate := &appsapi.Base{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      sub.Name,
+				Name:      fmt.Sprintf("%s-%s", sub.Name, sub.Namespace),
```

Comment:
Will the whole length of the name exceed the name length limit 253?

Reply:
Maybe, but this is a low-probability but multi-location issue. Such as
```
desc.Name = fmt.Sprintf("%s-generic", base.Name)
```
or
```
loc.Name = fmt.Sprintf("%s-%s-%s", base.Name, strings.ToLower(feedOrder.Feed.Kind), suffixName)
```
Do we need to solve this problem now?

Reply:
We can add an issue for tracking.